diff --git a/bashrc b/home/awildthorp/.bashrc
index 8d26792..e68bb2a 100644
--- a/bashrc
+++ b/home/awildthorp/.bashrc
@@ -136,7 +136,16 @@ fi
 # Set env variables
 export EDITOR=/bin/kak
 export GOPATH=$HOME/code/go
+export GOGIT=$GOPATH/src/github.com/aThorp96/
 export PATH=$PATH:$GOPATH/bin
 
+if ! pgrep -u "$USER" ssh-agent > /dev/null; then
+    ssh-agent > ~/.ssh-agent-thing
+    ssh-add -t 1h
+fi
+if [[ "$SSH_AGENT_PID" == "" ]]; then
+    eval "$(<~/.ssh-agent-thing)" > /dev/null
+fi
+
 #Run fish
 fish
